tt_metal/impl/buffers/circular_buffer_types.hpp

```diff
@@ -68,7 +68,7 @@ class CircularBufferConfig {
                 this->total_size_,
                 page_size);
         }
-        if (page_size % sizeof(uint32_t) != 0) {
+        if (page_size % 16 != 0) {
             TT_THROW("Page size must be divisible by sizeof(uint32_t) because buffers holds uint32_t values");
```

Comment:
Thanks for fixing. Please fix the throw message as well otherwise lgtm if pipelines pass